amz_review_scraper/models.py

```diff
@@ -12,6 +12,9 @@ class Item(db.Model):
     def save(self):
         db.session.add(self)
 
```

Comment:
Also its maybe not necessary now, but maybe if you added more, you would want to separate these into their own files, in a models folder.